### PR TITLE
fixes #378 Add another jwt verifier for tokens without scopes

### LINKED ISSUE #378: Add another jwt verifier for tokens without scopes
It allows path prefix to validate tokens from two different providers and one of them can be without scope like a transmit token.

### PR DESCRIPTION
**Pull request by stevehu**

### CODE REVIEW

File: openapi-security/src/main/java/com/networknt/openapi/UnifiedSecurityHandler.java [hunk 1]
```diff
@@ -106,43 +116,107 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
...
+                                // first to identify the token type.
+                                boolean isJwtOrSjwt = StringUtils.isJwtToken(token);
+                                if(logger.isTraceEnabled()) logger.trace("Bearer token is jwt or sjwt = {}", isJwtOrSjwt);
+                                if(isJwtOrSjwt) {
+                                    // if sjwt is true, need to parse the token to see if it has scope or scp claim to identify if it is a jwt or sjwt.
+                                    if(pathPrefixAuth.isSjwt()) {
+                                        if(logger.isTraceEnabled()) logger.trace("SJWT is enabled.");
+                                        // check if jwt is true.
+                                        if(pathPrefixAuth.isJwt()) {
+                                            if(logger.isTraceEnabled()) logger.trace("JWT is enabled along with swt.");
```
Comment (DiogoFKT) [inline openapi-security/src/main/java/com/networknt/openapi/UnifiedSecurityHandler.java:138]: Trace msg should be `JWT is enabled along with SJWT.`

File: openapi-security/src/main/java/com/networknt/openapi/SimpleJwtVerifyHandler.java [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+                        for(Map.Entry<String, String> entry: config.getPassThroughClaims().entrySet()) {
+                            String key = entry.getKey();
+                            String header = entry.getValue();
+                            Object value = claims.getClaimValue(key);
+                            if(logger.isTraceEnabled()) logger.trace("pass through header {} with value {}", header, value);
+                            headerMap.put(new HttpString(header), value.toString());
+                        }
+                    }
+                    if (logger.isTraceEnabled())
+                        logger.trace("complete JWT verification for request path = {}", exchange.getRequestURI());
```
Comment (DiogoFKT) [inline openapi-security/src/main/java/com/networknt/openapi/SimpleJwtVerifyHandler.java:173]: For consistency let's replace `JWT` for `SJWT`

File: openapi-security/src/main/java/com/networknt/openapi/UnifiedSecurityHandler.java (hunk 1 above)
Comment (stevehu) [reply openapi-security/src/main/java/com/networknt/openapi/UnifiedSecurityHandler.java:138]: Thanks for the catch. It is updated.

File: openapi-security/src/main/java/com/networknt/openapi/SimpleJwtVerifyHandler.java (hunk 2 above)
Comment (stevehu) [reply openapi-security/src/main/java/com/networknt/openapi/SimpleJwtVerifyHandler.java:173]: Thanks. It is updated.

**Pull request merged**